corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -131,6 +133,28 @@ hqDefine("geospatial/js/geospatial_map", [
                 });
             });
 
+            self.setDisbursementParameters = function (parameters) {
+                var parametersList = [
+                    {name: gettext("Max cases per user"), value: parameters.max_cases_per_user},
+                    {name: gettext("Min cases per user"), value: parameters.min_cases_per_user}
+                ];
+
+                if (parameters.max_case_distance) {
+                    const maxCaseDistanceParamValue = `${parameters.max_case_distance} km`;
+                    parametersList.push({name: gettext("Max distance to case"), value: maxCaseDistanceParamValue});
+                }
+
+                if (parameters.max_case_travel_time) {
+                    const travelParamValue = `${parameters.max_case_travel_time} ${gettext("minutes")}`;
```

Comment:
nit: Minutes?

Reply:
Do you mean I should capitalize it?

Reply:
I think both are okay, totally upto how its looking in the UI and preference.